templates/accounts/login.html

```diff
@@ -29,7 +29,8 @@ <h1>Sign In to {{ site_name }}</h1>
         </div>
         {% endif %}
 
-        <div class="row">
+        <div class="row" style="display: flex; justify-content: center;">
```

Comment:
Please move the inline CSS to a CSS file.

Reply:
So given its just two rules, I tried implementing this with bootstrap classes but bootstrap has flexbox support by default until v4+. I tried a drop-in replacement but that didn't help either.